main_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"net"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUsage(t *testing.T) {
+	w := bytes.NewBuffer(nil)
+	flag.CommandLine.SetOutput(w)
+	usage()
+	opts := `[-p <port>] [-a] [<dir>]`
```

Comment:
hmmm. i wonder if we (you) should order the options alphabetically?

Reply:
ok, done.

Reply:
you would have never picke on that without my usage test 👅

Reply:
no. and now you need to change it two places. perhaps you will consider some of your life (testing) choices. :tongue: